packages/cxl-ui/scss/cxl-navigation.scss

```diff
@@ -0,0 +1,75 @@
+@use "~@conversionxl/cxl-lumo-styles/scss/mq";
+
+:host {
+  --lumo-clickable-cursor: pointer;
+
+  ::part(menu-bar-button) {
+    color: var(--lumo-shade);
+    margin-inline: var(--lumo-space-s);
+  }
+
+  ::part(overflow-button) {
+    margin-left: auto;
+  }
+
+  nav {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    max-width: 100vw;
+    padding: 0 var(--lumo-space-m);
+
+    &[minimal] {
+      vaadin-menu-bar {
+        max-width: 64px;
+      }
```

Comment:
only affects `cxl-marketin-nav[minimal]`

Reply:
In the old component, the `minimal` attribute is used to hide search and menu button in the mobile layout. I was using the same attribute to enforce some mobile layout options, because `vaadin-menu-bar` tries to always show the first items. This is what it looks like without restricting the width:
   
![image](https://github.com/conversionxl/aybolit/assets/1887825/02156af6-ab00-47c5-a23e-fc7c730d1c37)

But that was an oversight on my part. I changed it to target :not([wide]), and minimal will do the exact same thing as it does on the old component, which is to look like this:
![image](https://github.com/conversionxl/aybolit/assets/1887825/1616d5d6-f72f-4660-aa12-9a67118b1456)